src/lib.rs

```diff
@@ -767,6 +767,38 @@ impl Context {
         }
     }
 
+    pub fn pcr_read(
+        &mut self,
+        pcr_selection: &TPML_PCR_SELECTION,
+    ) -> Result<(UINT32, TPML_PCR_SELECTION, TPML_DIGEST)> {
```

Comment:
It's ok for now since these params look fairly awkward to Rustify, but in the future the plan'll be to make these Rust-native (as per my comment on the other PR!). Just as a heads-up